include/amy/detail/value_cast.hpp

```diff
@@ -2,8 +2,6 @@
 #define __AMY_DETAIL_VALUE_CAST_HPP__
 
 #include <amy/sql_types.hpp>
-
```

Comment:
Please add back this newline. One coding convention in Amy is that, `amy/detail` headers, `amy/` headers, and 3rd-party headers are put into different groups separated by a newline, and file inclusions in each group are sorted in lexical order.